app/main/helpers/search_summary_manifest.yml

```diff
@@ -104,13 +104,35 @@
     id: mobile devices
   - preposition: an
     id: API
+
 - id: Service usage metrics
   labelPreposition: where usage metrics are provided through
   conjunction: and
 
-- id: Connected public sector networks
```

Comment:
Sorry if my Slack response didn't make it clear. The preferred name for this filter is "Connected public sector networks"